### ISSUE TITLE
Cannot parse quantity error

### ISSUE THREAD

**Issue by HackerMD** [uzh/gc3pie#607]
Got this error when running locally:

```
2017-02-09 12:08:43 | DEBUG    | gc3.gc3libs                              | Error getting status of application '67400': ValueError: Cannot parse quantity '%cpu %mem acflag acflg args blocked caught comm command cpu cputime etime f flags gid group ignored inblk inblock jobc ktrace ktracep lim login logname lstart majflt minflt msgrcv msgsnd ni nice nivcsw nsignals nsigs nswap nvcsw
nwchan oublk oublock p_ru paddr pagein pcpu pending pgid pid pmem ppid pri pstime putime re rgid rgroup rss ruid ruser sess sig sigmask sl start stat state stime svgid svuid tdev time tpgid tsess tsiz tt tty ucomm uid upr user usrpri
utime vsize vsz wchan wq wqb wql wqr xstatseconds'
Traceback (most recent call last):
  File "/Users/mdh/gc3pie/gc3libs/core.py", line 442, in __update_application
    state = lrms.update_job_state(app)
  File "/Users/mdh/gc3pie/gc3libs/backends/shellcmd.py", line 764, in update_job_state
    elapsed = Duration(stdout2.strip() + 'seconds')
  File "/Users/mdh/gc3pie/gc3libs/quantity.py", line 949, in __new__
    return _Quantity.__new__(cls, val, unit, name)
  File "/Users/mdh/gc3pie/gc3libs/quantity.py", line 272, in __new__
    new = cls._new_from_string(val)
  File "/Users/mdh/gc3pie/gc3libs/quantity.py", line 992, in _new_from_string
    return super(Duration, cls)._new_from_string(val)
  File "/Users/mdh/gc3pie/gc3libs/quantity.py", line 304, in _new_from_string
    amount, unitname = _split_amount_and_unit(val, allow=cls._UNITS)
  File "/Users/mdh/gc3pie/gc3libs/quantity.py", line 98, in _split_amount_and_unit
    raise ValueError("Cannot parse quantity '%s'" % val)
ValueError: Cannot parse quantity '%cpu %mem acflag acflg args blocked caught comm command cpu cputime etime f flags gid group ignored inblk inblock jobc ktrace ktracep lim login logname lstart majflt minflt msgrcv msgsnd ni nice nivcsw nsignals nsigs nswap nvcsw
nwchan oublk oublock p_ru paddr pagein pcpu pending pgid pid pmem ppid pri pstime putime re rgid rgroup rss ruid ruser sess sig sigmask sl start stat state stime svgid svuid tdev time tpgid tsess tsiz tt tty ucomm uid upr user usrpri
utime vsize vsz wchan wq wqb wql wqr xstatseconds'
2017-02-09 12:08:43 | WARNING  | gc3.gc3libs                              | Ignored error in Core.update_job_state(): Cannot parse quantity '%cpu %mem acflag acflg args blocked caught comm command cpu cputime etime f flags gid group ignored inblk inblock jobc ktrace ktracep lim login logname lstart majflt minflt msgrcv msgsnd ni nice nivcsw nsignals nsigs nswap nvcsw
nwchan oublk oublock p_ru paddr pagein pcpu pending pgid pid pmem ppid pri pstime putime re rgid rgroup rss ruid ruser sess sig sigmask sl start stat state stime svgid svuid tdev time tpgid tsess tsiz tt tty ucomm uid upr user usrpri
utime vsize vsz wchan wq wqb wql wqr xstatseconds'
2017-02-09 12:08:43 | DEBUG    | gc3.gc3libs                              | (Original traceback follows.)
Traceback (most recent call last):
  File "/Users/mdh/gc3pie/gc3libs/core.py", line 451, in __update_application
    raise ex
ValueError: Cannot parse quantity '%cpu %mem acflag acflg args blocked caught comm command cpu cputime etime f flags gid group ignored inblk inblock jobc ktrace ktracep lim login logname lstart majflt minflt msgrcv msgsnd ni nice nivcsw nsignals nsigs nswap nvcsw
nwchan oublk oublock p_ru paddr pagein pcpu pending pgid pid pmem ppid pri pstime putime re rgid rgroup rss ruid ruser sess sig sigmask sl start stat state stime svgid svuid tdev time tpgid tsess tsiz tt tty ucomm uid upr user usrpri
utime vsize vsz wchan wq wqb wql wqr xstatseconds'
```

**Comment by riccardomurri** [comment]
"running locally" -> on MacOSX?

**Comment by HackerMD** [comment]
yes

**Comment by riccardomurri** [comment]
Can you please post the output of this command on your MacOSX system?

    ps -p $$ -o etimes=

**Comment by HackerMD** [comment]
```
ps: etimes: keyword not found
ps: no valid keywords; valid keywords:
%cpu %mem acflag acflg args blocked caught comm command cpu cputime etime f flags gid group ignored inblk inblock jobc ktrace ktracep lim login logname lstart majflt minflt msgrcv msgsnd ni nice nivcsw nsignals nsigs nswap nvcsw
nwchan oublk oublock p_ru paddr pagein pcpu pending pgid pid pmem ppid pri pstime putime re rgid rgroup rss ruid ruser sess sig sigmask sl start stat state stime svgid svuid tdev time tpgid tsess tsiz tt tty ucomm uid upr user usrpri
utime vsize vsz wchan wq wqb wql wqr xstat
```

**Comment by riccardomurri** [comment]
Try this instead::

    ps -p $$ -o etime=

**Comment by HackerMD** [comment]
$ ps -p $$ -o etime=
```
02:32:21
```

Looks good :)

**Comment by riccardomurri** [comment]
One thing more: if your laptop hasn't been rebooted in a few days (or if you can find a Mac which hasn't), can you please run::

    ps -p 1 -o etime=

**Comment by HackerMD** [comment]
My Mac hasn't been rebooted in quite some time - apparently...

$ ps -p 1 -o etime=   
```                                                                                                                                                                                      
25-00:31:05
```

**Comment by riccardomurri** [comment]
Should be fixed now in commit c22809b -- please test and re-open if you get the same error.

Thanks for reporting!
